src/cargo/ops/cargo_rustc/mod.rs

```diff
@@ -526,7 +527,7 @@ fn add_plugin_deps(rustc: &mut ProcessBuilder,
     let mut search_path = env::split_paths(&search_path).collect::<Vec<_>>();
     for id in build_scripts.plugins.iter() {
         let key = (id.clone(), Kind::Host);
-        let output = build_state.get(&key).chain_error(|| {
+        let output = build_state.get(&key).ok_or_else(|| {
             internal(format!("couldn't find libs for plugin dep {}", id))
```

Comment:
Ideally we could get rid of `internal` and `human` as function (I think `error-chain` just supports strings strings in these closures), do you think that'd be feasible? Or maybe happen in a future PR?